client/src/project/Project.present.js

```diff
@@ -263,6 +263,65 @@ function ProjectViewHeaderOverviewDescription({ settingsReadOnly, description, s
   return null;
 }
 
+const ProjectSuggestActions = (props) => {
+  useEffect(() => {
+    props.fetchDatasets();
+  }, []); // eslint-disable-line
+  const commits = props.projectCoordinator.get("commits");
+  if (!commits.fetching)
+    props.projectCoordinator.fetchCommits();
+
+  const loadingCommits = !commits.fetched;
+  const loadingCommitsReadme = isRequestPending(props, "commitsReadme");
+  const loadingDatasets = typeof (props.core?.datasets) === "string";
+
+  if (loadingCommits || loadingCommitsReadme || loadingDatasets) return "";
+
+  const cantTotalCommits = props.model?.get("project.commits.list")?.length ?? 0;
+  const hasDatasets = props.core?.datasets?.length > 0;
+  const countCommitsReadme = props.projectCoordinator.get("data.readme.countCommits");
+
+  if (cantTotalCommits > 4 || (countCommitsReadme > 1 && hasDatasets)) return "";
+
+  const gitlabIDEUrl = props.externalUrl !== "" && props.externalUrl.includes("/gitlab/") ?
+    props.externalUrl.replace("/gitlab/", "/gitlab/-/ide/project/") : null;
+  const addReadmeUrl = `${gitlabIDEUrl}/edit/${props.core.default_branch}/-/README.md`;
+
+  const suggestionReadme = countCommitsReadme > 1 ? null
```

Comment:
This should be > 0, I think.

```suggestion
  const suggestionReadme = countCommitsReadme > 0 ? null
```

Reply:
Thanks for the revision, it should be 1 because the automatic initial commit should not be included.
The reason the suggestion component was not updating after a new commit is because it was being calculated when the project was loaded, I have modified it to calculate the number of readme commits once the total amount of commits in the project changes.

Reply:
Yes, you are correct!